Updating LabKey fork with changes to wnprc21.11-snapshot branch

dnicolalde opened the pull request:
#### Rationale
Updating LabKey fork with some changes done by WNPRC developers

#### Related Pull Requests
* <!-- list of links to related pull requests (replace this comment) -->

#### Changes
* <!-- list of descriptions of changes that are worth noting (replace this comment) -->

gottfred committed:
added pattern matching for rhesus macaque ids to pull ids from strings such as r#####a

dnicolalde committed:
Merge pull request #233 from WNPRC-EHR-Services/21.3w_fb_staging

Catching up SNAPSHOT with staging

gottfred committed:
format total_reads column to have comma thousandth separators

gottfred committed:
add calculated column to demographics table for mGAP URL

dnicolalde committed:
Modifying insert Husbandry_frequency insert and adding dropdown configuration for Implant cleaning

gottfred committed:
create a link to the external mGAP system from demographics mgapIds column

dnicolalde committed:
Fixing problem with waterMonitoringAccess on production.

dnicolalde committed:
Merge pull request #234 from WNPRC-EHR-Services/21.3w_fb_staging

Modifying insert Husbandry_frequency insert

gottfred committed:
remove unused code, clean up sql query, stage new URL for when it's ready

gottfred committed:
update animal abstract code to more gracefully handle an odd number of columns, as well as filtering to only show non-hidden columns

gottfred committed:
Merge branch 'wnprc21.3-SNAPSHOT' into 21.3w_fb_mgap_integration

gottfred committed:
fix so that large purchase notifications don't get emailed to all site admins. per labkey ticket 44448

dnicolalde committed:
Adding system property to fix problem with Log4J

dnicolalde committed:
Merge pull request #235 from WNPRC-EHR-Services/wnprc21.3-SNAPSHOT

Wnprc21.3 snapshot

dnicolalde committed:
Updating secondary labkey instance for log4J security issue.

dnicolalde committed:
Adding default values to .env

dnicolalde committed:
Merge pull request #236 from WNPRC-EHR-Services/21.3w_fb_staging

Updating 21.7 with latest development from 21.3 staging

dnicolalde committed:
Merge branch 'wnprc21.7-SNAPSHOT' of https://github.com/WNPRC-EHR-Services/wnprc-modules into wnprc21.7-SNAPSHOT

dnicolalde committed:
Updating version of Postgres

dnicolalde committed:
Merge pull request #237 from LabKey/release21.7-SNAPSHOT

Catching up WNPRC-EHR SNAPSHOT21.7 from LabKey WNPRC repo

dnicolalde committed:
Update LabKey version to 21.7

dnicolalde committed:
Merge branch 'wnprc21.7-SNAPSHOT' of https://github.com/WNPRC-EHR-Services/wnprc-modules into wnprc21.7-SNAPSHOT

gottfred committed:
update primary key to use 3 columns

gottfred committed:
update to new mGAP url and fix bug so that animal id case doesn't matter for matching to calculated column

gottfred committed:
add mGAP sequence type to demographics calculated column as well as animal abstract

gottfred committed:
disable editing of the date field for pregnancies. this fixes a bug where pregnancy outcomes and ultrasounds can become unlinked from the pregnancy due to an lsid change

gottfred committed:
remove admin user from list of users that should receive notification emails

dnicolalde committed:
Update to watermonitoring to update waterAmounts from dataset.

dnicolalde committed:
Merge remote-tracking branch 'origin/21.3w_fb_staging' into 21.3w_fb_staging

gottfred committed:
Revert "remove admin user from list of users that should receive notification emails"

This reverts commit 274f062d60eac88e8c956b2aea9ffac6c922ac41.

gottfred committed:
Merge branch '21.3w_fb_staging' of ssh://github.com/WNPRC-EHR-Services/wnprc-modules into 21.3w_fb_staging

gottfred committed:
remove current user from list of users that should receive notification emails for >=000 purchase approvals and rejections

gottfred committed:
Revert "remove current user from list of users that should receive notification emails for >=000 purchase approvals and rejections"

This reverts commit c360bd9c0ec010f48ae128e3ede2646460b3c32c.

gottfred committed:
Revert "Revert "remove current user from list of users that should receive notification emails for >=000 purchase approvals and rejections""

This reverts commit 51278a1a72e516936b8f823506a8f925c529a761.

dnicolalde committed:
Update to pedigree ETL for U24 collaboration

dnicolalde committed:
Adding POST to method for other functions inside the waterCalendar.jsp

dnicolalde committed:
Allowing chairing to be updated outside the LabWaterForm.

dnicolalde committed:
Removing edit date from grid in the Enter Treatment - Water form.

dnicolalde committed:
Adding waterOrderObjectId to the waterObject to only include waterAmounts associated with the WaterOrder.

dnicolalde committed:
Using waterOrderObjectId to group the waterAmounts to the waterOrder where it came from.

dnicolalde committed:
Improving error messaging handling in model window for waterAmounts in the future

dnicolalde committed:
Merge pull request #238 from WNPRC-EHR-Services/21.3w_fb_staging

Improving UI on the water monitoring and allow to edit records on the water system

dnicolalde committed:
Fixing waterAmount QC status change and requiring project in the Lab water form.

gottfred committed:
Merge pull request #239 from WNPRC-EHR-Services/21.3w_fb_staging

Fixing waterAmount QC status change and requiring project

labkey-jeckels committed:
Bump all schema versions to 22.000 (#147) (#240)

Co-authored-by: F. Daniel Nicolalde <fdnicolalde@wisc.edu>
Co-authored-by: Gabriel Gottfredsen <gabriel.gottfredsen@gmail.com>
Co-authored-by: Adam Rauch <adam@labkey.com>

labkey-jeckels committed:
Bump all schema versions to 22.000 (#147) (#240)

Co-authored-by: F. Daniel Nicolalde <fdnicolalde@wisc.edu>
Co-authored-by: Gabriel Gottfredsen <gabriel.gottfredsen@gmail.com>
Co-authored-by: Adam Rauch <adam@labkey.com>

gottfred committed:
update dataset column to correct name so that it will show by default in animal full history

gottfred committed:
update dataset column to correct name so that it will show by default in animal full history

dnicolalde committed:
Fixing couple of issue with 21.7 from import.

dnicolalde committed:
Revert "Fixing couple of issue with 21.7 from import."

This reverts commit bffecddd

dnicolalde committed:
Revert "Fixing couple of issue with 21.7 from import."

This reverts commit bffecddd

dnicolalde committed:
Merge pull request #241 from WNPRC-EHR-Services/wnprc21.3-SNAPSHOT

Catch up 21.7 branch with recent changes

dnicolalde committed:
Adding attributes to WNPRC repo

dnicolalde committed:
Modifiying dbUtils code based on conversation with LabKey on Jan 7, 2022

dnicolalde committed:
Catching a change the fork sync overwrite.

gottfred committed:
Merge branch '21.3w_fb_staging' into 21.3w_fb_mgap_integration

gottfred committed:
correcting issue tracker links, and updating some wording on the main EHR page as per IDS issue #13643

gottfred committed:
missed an old issue tracker link in the last update

dnicolalde committed:
Updating version on README file

dnicolalde committed:
Merge branch '21.3w_fb_staging' of https://github.com/WNPRC-EHR-Services/wnprc-modules into 21.3w_fb_staging

dnicolalde committed:
Removing one space to test DockerHub

dnicolalde committed:
Update to default branch to avoid problem with DockerHub

dnicolalde committed:
Removing old version of LabKey from Docker Hub

gottfred committed:
update a couple more issue tracker links

gottfred committed:
Merge branch '21.3w_fb_staging' of ssh://github.com/WNPRC-EHR-Services/wnprc-modules into 21.3w_fb_staging

dnicolalde committed:
Merge pull request #242 from WNPRC-EHR-Services/wnprc21.7-SNAPSHOT

Updating 21.11 with all changes from 21.7 and 21.3 fro wnprcModules

dnicolalde committed:
Updating Dockerfile for LabKey build and adding .gitattributes

dnicolalde committed:
Removing space from wnprcLabs.html and adding the file to .gitattributes

dnicolalde committed:
Merge branch '21.3w_fb_staging' of https://github.com/WNPRC-EHR-Services/wnprc-modules into 21.3w_fb_staging

dnicolalde committed:
Updating default version for Dockerfile

dnicolalde committed:
Removing some old svn information.

dnicolalde committed:
Update to pedigree etl for u24 project

dnicolalde committed:
Updating version of LabKey in the Dockerfile

gottfred committed:
Merge pull request #244 from LabKey/release21.11-SNAPSHOT

Fix for test timing issues. (#152)

gottfred committed:
fixed spacing

gottfred committed:
update tomcat to latest point release of version 9 (9.0.54 -> 9.0.56)

gottfred committed:
udpated java version from 15 to 17

dnicolalde committed:
Removing older svn reference

gottfred committed:
Merge branch '21.3w_fb_staging' into wnprc21.11-SNAPSHOT

gottfred committed:
update README with default values for 21.11

dnicolalde committed:
Small change with docker link

dnicolalde committed:
Merge pull request #243 from WNPRC-EHR-Services/21.3w_fb_staging

Catching up staging with SNAPSHOT for 21.3

dnicolalde committed:
Adding watermonitoring_access definition

gottfred committed:
add mgap_sequence_datasets table info into wnprc.xml file

gottfred committed:
Merge branch 'wnprc21.11-SNAPSHOT' of ssh://github.com/WNPRC-EHR-Services/wnprc-modules into wnprc21.11-SNAPSHOT

gottfred committed:
determine which build to pull from teamcity based on the git branch (source branch) name

dnicolalde committed:
Merge pull request #247 from LabKey/release21.11-SNAPSHOT

Fixes for test timing issue (#154)

dnicolalde committed:
Update ehrcron to following branch naming and docker hub tags.

dnicolalde committed:
Correcting a typo of the readme file

dnicolalde committed:
Updating readme file.

dnicolalde committed:
Adding sort and qview for waterTotalByDate

dnicolalde committed:
Updating perl and also version of debian

dnicolalde committed:
Fixing proplem with imklog

dnicolalde committed:
adding source to imklog problem

dnicolalde committed:
adding ehr .ini to the ignore

dnicolalde committed:
Adding enviromental variables to ehrcron

dnicolalde committed:
Adding new variable to default.env

dnicolalde committed:
fix to new .env variable

dnicolalde committed:
fixing typo on the default.env file

dnicolalde committed:
Removing pguser from .ini files and moving to .env file

dnicolalde committed:
Adding pg backup user and adding ARG prod_version to modify the crontab for prod server

dnicolalde committed:
removing old split for db names

dnicolalde committed:
Adding LabKey version to .env and way to defined prod in the .env

dnicolalde committed:
Adding a hook to use the ARG on docker hub

dnicolalde committed:
Adding environmental variables to '/etc/environment' so that cron job have access to the variables

dnicolalde committed:
Removing LK_BASE_URL from cron jobs the system will use environmental variables.

dnicolalde committed:
Printing names of dbs to backup.

dnicolalde committed:
Removing location of postgres to .env and the docker-compose

dnicolalde committed:
Fixes to help Safari resolve dates from the database.

dnicolalde committed:
Adding water forms to render back for editing existing records.

gottfred reviewed (approved):
Appears to be all of our recent updates

dnicolalde committed:
Improving editing water monitoring system by adding away to edit records from taskid link.

dnicolalde commented:
@labkey-jeckels -- Going to commit this to SNAPSHOT 21.11, it has some of the change we have worked for 21.3

gottfred committed:
update docker hook to be version-agnostic

gottfred committed:
small change to test docker hub

gottfred committed:
update docker hook to be version-agnostic

dnicolalde committed:
Fixing several problems with watermonitoring system, issue# 3297

labkey-martyp committed:
Add URL sync with dataEntry tab panel (#161)

- Use similar restoreFromUrl as EHR but updated for BootstrapTabPanel
- Add changeHandler to update location on tab selection

dnicolalde committed:
Small change to the EHR controller to better work with Ext4 pages.

dnicolalde committed:
Merge remote-tracking branch 'origin/21.3w_fb_staging' into 21.3w_fb_waterImprovements

dnicolalde committed:
Adding provide fruit to animal Full history.

gottfred committed:
add a line number for each item in an order

gottfred committed:
change Special instructions to Comments/Special instructions on purchasing request page

gottfred committed:
make the Request No. column of purchasing line items link out to the actual request

gottfred committed:
add link to show my purchased items list

dnicolalde committed:
Adding weight to the waterTotalByDate and expandin the query to cover 40 day in the past.

dnicolalde committed:
Change color of completed water events in the calendar.

dnicolalde committed:
Changing sort order to descending order.

dnicolalde committed:
Merge pull request #255 from WNPRC-EHR-Services/21.3w_fb_waterImprovements

21.3w fb water improvements

dnicolalde committed:
Creating validation to allow water system user to modify their own water orders.

dnicolalde committed:
Adding TODO for continue in the water improvement branch.

dnicolalde committed:
Removing unnecesary comments.

dnicolalde committed:
Removing unnecesary code and commented lines.

dnicolalde committed:
Solving the problem with all water totals turning into red.

dnicolalde committed:
Merge pull request #251 from WNPRC-EHR-Services/21.3w_fb_staging

update docker hook to be version-agnostic and other changes to the water monitoring system

dnicolalde committed:
Fixing problem with calendar view.

dnicolalde committed:
Implementing waterOrder permissions based on watermonitoring_access table

dnicolalde committed:
Merge pull request #257 from WNPRC-EHR-Services/21.3w_fb_staging

Fixing problem with calendar view.

labkey-bpatel committed:
Reset order received quantity when reordering. (#164)

Reset 'quantity received' when reordering, which allows for the qc state to be 'Review Pending'.

dnicolalde committed:
Fixing problem with wnprcFullHistory.sql

dnicolalde committed:
Merge pull request #259 from WNPRC-EHR-Services/21.3w_fb_staging

Fixing problem with wnprcFullHistory.sql

labkey-teamcity committed:
Merge 21.7 to 21.11

dnicolalde committed:
Improving ChargeItemField.js to remove items from the dropdown base on the date.

dnicolalde committed:
Further improvements and consolidating the select function.

dnicolalde committed:
Remove console logs and not used code

dnicolalde committed:
Merge branch '21.3w_fb_staging' into 21.3w_fb_waterImprovements

dnicolalde committed:
Enabling animal care to see water order in the calendar view.

dnicolalde committed:
Allowing animal care to see water calendar but restricting edit.

dependabot[bot] committed:
Bump minimist from 1.2.5 to 1.2.6 in /WNPRC_Purchasing

Bumps [minimist](https://github.com/substack/minimist) from 1.2.5 to 1.2.6.
- [Release notes](https://github.com/substack/minimist/releases)
- [Commits](https://github.com/substack/minimist/compare/1.2.5...1.2.6)

---
updated-dependencies:
- dependency-name: minimist
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dnicolalde committed:
Using docker tag to determine the teamcity installer

dnicolalde committed:
Changes to package.json for WNPRC modules

dnicolalde committed:
Changes to webpack and gradle

dnicolalde committed:
Committing package-lock to see if TeamCity starts building

dnicolalde committed:
Adding package-lock to see if TeamCity starts building

dnicolalde committed:
Adding version to hook file

dnicolalde committed:
Modifying package-lock.json after deleting the node_modules folder.

dnicolalde committed:
Adding single transaction to input water amounts from calendar.

dnicolalde committed:
Using TypeScript to save the water amount and task into one transaction.

dnicolalde committed:
Rehydrating npm for wnprc-ehr and WHNPRC_EHR modules

dnicolalde committed:
Merge branch '21.3w_fb_fixingNpm' of https://github.com/WNPRC-EHR-Services/wnprc-modules into 21.3w_fb_fixingNpm

 Conflicts:
	package-lock.json

labkey-nicka committed:
WNPRC_Purchasing: Package Updates
- rehydrate package-lock.json

labkey-nicka committed:
WNPRC_Purchasing: update script commands

labkey-nicka committed:
Update .gitignore for @labkey/build paths

dnicolalde committed:
Moving package.json inside the primateid module and further modifying build process for the wnprc-modules. Rehydrating package-lock.json for primateid, WNPRC_EHR and WNPRC_Purchasing

dnicolalde committed:
Removing StoryBook from package.json file for primateid

dnicolalde committed:
Fixing package.json and building from npm 6.13.4 except for wnprc modules which uses 8.1.2

dnicolalde committed:
Submitting the .npmrc files to allow using labkey artifactory

csebranek committed:
Patch for fixing performedby field in obs dataset see LK ticket 45200

csebranek committed:
Fix npm clean scripts

csebranek committed:
Add back in javascript gradle file

csebranek committed:
Add webpack-cli in an attempt to fix teamcity build for WNPRC_EHR npm modules

csebranek committed:
Strip down primate id module and rehyrdate some package-lock files

dnicolalde committed:
Merge pull request #262 from WNPRC-EHR-Services/21.3w_fb_fixingNpm

21.3w fb fixing npm

csebranek committed:
use the primateid npm repo, fixes webpack build

dependabot[bot] committed:
Bump async from 2.6.1 to 2.6.4 in /AnimalRequests

Bumps [async](https://github.com/caolan/async) from 2.6.1 to 2.6.4.
- [Release notes](https://github.com/caolan/async/releases)
- [Changelog](https://github.com/caolan/async/blob/v2.6.4/CHANGELOG.md)
- [Commits](https://github.com/caolan/async/compare/v2.6.1...v2.6.4)

---
updated-dependencies:
- dependency-name: async
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump lodash-es from 4.17.15 to 4.17.21 in /AnimalRequests

Bumps [lodash-es](https://github.com/lodash/lodash) from 4.17.15 to 4.17.21.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.21)

---
updated-dependencies:
- dependency-name: lodash-es
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

csebranek committed:
Adjust obs trigger script to save displayName only LK ticket 45200

csebranek committed:
remove ts reference to primateid in the wnprc_ehr module

dnicolalde committed:
Merge remote-tracking branch 'origin/21.3w_fb_staging' into 21.3w_fb_waterImprovements

# Conflicts:
#	WNPRC_EHR/.npmrc
#	WNPRC_EHR/package-lock.json
#	WNPRC_EHR/package.json
#	WNPRC_EHR/webpack.config.js
#	package-lock.json

csebranek committed:
let webpack build primateid not tsc

csebranek committed:
Revert userid cacheing attempt on obs trigger script until it is working, looking up userids is not that expensive

gottfred committed:
Merge pull request #272 from LabKey/release21.11-SNAPSHOT

Release21.11 snapshot

dnicolalde committed:
Adding date to the animalwithoutentries sql report.

gottfred committed:
Merge branch 'wnprc21.11-SNAPSHOT' into 21.11_fb_purchasing_line_item_numbers

dnicolalde committed:
Modifying water monitoring to improve display of notification and group by project number.

dnicolalde committed:
Updating package-lock.json for WNPRC_EHR module

dnicolalde committed:
Saving typescript file, but using LabKey.query to save waterAmount and task in one transaction.

dnicolalde committed:
Adding new times to send the notification.

dnicolalde committed:
Merge remote-tracking branch 'upstream/21.3w_fb_staging' into 21.11w_fb_mergingLatest21.3

dnicolalde committed:
Removing additional import for HashedMap.

dnicolalde committed:
Merge pull request #273 from WNPRC-EHR-Services/21.3w_fb_waterImprovements

21.3w fb water improvements

dnicolalde committed:
Updating package-lock.json for primateid and WNPRC_EHR modules

dnicolalde committed:
Merge branch '21.3w_fb_staging' into 21.11w_fb_mergingLatest21.3

# Conflicts:
#	WNPRC_EHR/package-lock.json
#	WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/husbandry/WaterCalendar.jsp

dnicolalde committed:
Changing label of notification.

dnicolalde committed:
Merge pull request #274 from WNPRC-EHR-Services/21.3w_fb_staging

Bringing change to SNAPSHOT

dnicolalde committed:
Rehydrating package-lock.json based on error on TeamCity

csebranek committed:
Delete .npmrc

primateid doesn't need @labkey npm repo

dnicolalde committed:
Fixing problem with notification table for water monitoring.

dnicolalde committed:
Merge pull request #276 from WNPRC-EHR-Services/21.3w_fb_staging

Fixing problem with notification table for water monitoring.

dnicolalde committed:
Merge pull request #277 from WNPRC-EHR-Services/wnprc21.3-SNAPSHOT

Wnprc21.3 snapshot

dnicolalde committed:
Merge pull request #275 from WNPRC-EHR-Services/21.11_fb_mergingLatest21.3

21.11 fb merging latest21.3

dnicolalde committed:
Adding some formatting for checks done for days in the pass for the WaterMonitoringAnimalWithOutEntriesNotification.java.

dnicolalde committed:
Merge branch '21.3w_fb_staging' into dependabot/npm_and_yarn/elliptic-6.5.4

dnicolalde committed:
push origin 21.3w_fb_staging from dependabot pull request
Merge branch 'dependabot/npm_and_yarn/elliptic-6.5.4' into 21.3w_fb_staging

dnicolalde committed:
Merge pull request #179 from WNPRC-EHR-Services/dependabot/npm_and_yarn/y18n-4.0.1

Bump y18n from 4.0.0 to 4.0.1

dnicolalde committed:
Merge pull request #187 from WNPRC-EHR-Services/dependabot/npm_and_yarn/ssri-6.0.2

Bump ssri from 6.0.1 to 6.0.2

dnicolalde committed:
Merge branch '21.3w_fb_staging' into dependabot/npm_and_yarn/underscore-1.12.1

dnicolalde committed:
Merge branch 'dependabot/npm_and_yarn/underscore-1.12.1' it push origin 21.3w_fb_staging

dnicolalde committed:
Merge remote-tracking branch 'origin/21.3w_fb_staging' into 21.3w_fb_staging

dependabot[bot] committed:
Bump immer from 7.0.7 to 9.0.6 in /WNPRC_Purchasing

Bumps [immer](https://github.com/immerjs/immer) from 7.0.7 to 9.0.6.
- [Release notes](https://github.com/immerjs/immer/releases)
- [Commits](https://github.com/immerjs/immer/compare/v7.0.7...v9.0.6)

---
updated-dependencies:
- dependency-name: immer
  dependency-type: direct:development
...

Signed-off-by: dependabot[bot] <support@github.com>

dnicolalde committed:
Committing changes based on pull request #191

dnicolalde committed:
Merge branch 'dependabot/npm_and_yarn/lodash-4.17.21' into 21.3w_fb_stagingt push origin 21.3w_fb_staging based on pull request #191

dependabot[bot] committed:
Bump lodash-es from 4.17.15 to 4.17.21

Bumps [lodash-es](https://github.com/lodash/lodash) from 4.17.15 to 4.17.21.
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.15...4.17.21)

---
updated-dependencies:
- dependency-name: lodash-es
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump postcss from 7.0.27 to 7.0.39

Bumps [postcss](https://github.com/postcss/postcss) from 7.0.27 to 7.0.39.
- [Release notes](https://github.com/postcss/postcss/releases)
- [Changelog](https://github.com/postcss/postcss/blob/7.0.39/CHANGELOG.md)
- [Commits](https://github.com/postcss/postcss/compare/7.0.27...7.0.39)

---
updated-dependencies:
- dependency-name: postcss
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump urijs from 1.19.1 to 1.19.11

Bumps [urijs](https://github.com/medialize/URI.js) from 1.19.1 to 1.19.11.
- [Release notes](https://github.com/medialize/URI.js/releases)
- [Changelog](https://github.com/medialize/URI.js/blob/gh-pages/CHANGELOG.md)
- [Commits](https://github.com/medialize/URI.js/compare/v1.19.1...v1.19.11)

---
updated-dependencies:
- dependency-name: urijs
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump moment from 2.22.1 to 2.29.2

Bumps [moment](https://github.com/moment/moment) from 2.22.1 to 2.29.2.
- [Release notes](https://github.com/moment/moment/releases)
- [Changelog](https://github.com/moment/moment/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/moment/moment/compare/2.22.1...2.29.2)

---
updated-dependencies:
- dependency-name: moment
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump url-parse from 1.4.7 to 1.5.10

Bumps [url-parse](https://github.com/unshiftio/url-parse) from 1.4.7 to 1.5.10.
- [Release notes](https://github.com/unshiftio/url-parse/releases)
- [Commits](https://github.com/unshiftio/url-parse/compare/1.4.7...1.5.10)

---
updated-dependencies:
- dependency-name: url-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump ua-parser-js from 0.7.17 to 0.7.31

Bumps [ua-parser-js](https://github.com/faisalman/ua-parser-js) from 0.7.17 to 0.7.31.
- [Release notes](https://github.com/faisalman/ua-parser-js/releases)
- [Commits](https://github.com/faisalman/ua-parser-js/compare/0.7.17...0.7.31)

---
updated-dependencies:
- dependency-name: ua-parser-js
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dnicolalde committed:
Merge pull request #204 from WNPRC-EHR-Services/dependabot/npm_and_yarn/set-getter-0.1.1

Bump set-getter from 0.1.0 to 0.1.1

dnicolalde committed:
Merge pull request #212 from WNPRC-EHR-Services/dependabot/npm_and_yarn/path-parse-1.0.7

Bump path-parse from 1.0.5 to 1.0.7

dnicolalde committed:
Merge pull request #216 from WNPRC-EHR-Services/dependabot/npm_and_yarn/WNPRC_Purchasing/immer-9.0.6

Bump immer from 7.0.7 to 9.0.6 in /WNPRC_Purchasing

dnicolalde committed:
Merge pull request #248 from WNPRC-EHR-Services/dependabot/npm_and_yarn/ua-parser-js-0.7.31

Bump ua-parser-js from 0.7.17 to 0.7.31

dnicolalde committed:
Merge pull request #263 from WNPRC-EHR-Services/dependabot/npm_and_yarn/moment-2.29.2

Bump moment from 2.22.1 to 2.29.2

dependabot[bot] committed:
Bump moment from 2.24.0 to 2.29.2 in /AnimalRequests

Bumps [moment](https://github.com/moment/moment) from 2.24.0 to 2.29.2.
- [Release notes](https://github.com/moment/moment/releases)
- [Changelog](https://github.com/moment/moment/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/moment/moment/compare/2.24.0...2.29.2)

---
updated-dependencies:
- dependency-name: moment
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dnicolalde committed:
Merge pull request #264 from WNPRC-EHR-Services/dependabot/npm_and_yarn/AnimalRequests/moment-2.29.2

Bump moment from 2.24.0 to 2.29.2 in /AnimalRequests

dnicolalde committed:
Merge pull request #265 from WNPRC-EHR-Services/dependabot/npm_and_yarn/urijs-1.19.11

Bump urijs from 1.19.1 to 1.19.11

dnicolalde committed:
Merge pull request #270 from WNPRC-EHR-Services/dependabot/npm_and_yarn/AnimalRequests/async-2.6.4

Bump async from 2.6.1 to 2.6.4 in /AnimalRequests

dnicolalde committed:
Merge pull request #271 from WNPRC-EHR-Services/dependabot/npm_and_yarn/AnimalRequests/lodash-es-4.17.21

Bump lodash-es from 4.17.15 to 4.17.21 in /AnimalRequests

dnicolalde committed:
Merge pull request #278 from WNPRC-EHR-Services/dependabot/npm_and_yarn/lodash-es-4.17.21

Bump lodash-es from 4.17.15 to 4.17.21

dnicolalde committed:
Merge pull request #279 from WNPRC-EHR-Services/dependabot/npm_and_yarn/postcss-7.0.39

Bump postcss from 7.0.27 to 7.0.39

dnicolalde committed:
Merge pull request #280 from WNPRC-EHR-Services/dependabot/npm_and_yarn/url-parse-1.5.10

Bump url-parse from 1.4.7 to 1.5.10

dnicolalde committed:
Merge pull request #261 from WNPRC-EHR-Services/dependabot/npm_and_yarn/WNPRC_Purchasing/minimist-1.2.6

Bump minimist from 1.2.5 to 1.2.6 in /WNPRC_Purchasing

dnicolalde committed:
Applying changes made by dependabot into the primateId package.json, rehydrating package-lock.json files for all the modules using npm and removing package.json from the top folder.

dnicolalde committed:
Revert "Further improvements and consolidating the select function."

This reverts commit a3af855a6c9d27c5bd5d9ec1c68b5cdd3cd306d7.

dnicolalde committed:
Revert "Improving ChargeItemField.js to remove items from the dropdown base on the date."

This reverts commit b4da8a7f3f5f3e80daaba0d8eadb6c3e4c0772e6.

dnicolalde committed:
Merge pull request #282 from WNPRC-EHR-Services/21.3w_fb_staging

Catching changes from dependabot and reverts

dnicolalde committed:
Updating package-lock for resolve pull request #283

dnicolalde committed:
Merge branch 'wnprc21.11-SNAPSHOT' into wnprc21.3-SNAPSHOT

dnicolalde committed:
Merge branch 'wnprc21.3-SNAPSHOT' into wnprc21.11-SNAPSHOT

dnicolalde committed:
Replacing package-lock with newer versions

gottfred committed:
Merge branch 'wnprc21.11-SNAPSHOT' into wnprc21.7-SNAPSHOT

gottfred committed:
Merge pull request #284 from WNPRC-EHR-Services/wnprc21.7-SNAPSHOT

Wnprc21.7 snapshot

dnicolalde committed:
Update to load_database script and updating version of Rlabkey

dnicolalde committed:
Merge branch 'wnprc21.11-SNAPSHOT' of https://github.com/WNPRC-EHR-Services/wnprc-modules into wnprc21.11-SNAPSHOT

dnicolalde committed:
Adding environmental variables to be used by .netrc on ran time.

dnicolalde committed:
Adding environmental variables to the develop subfolder

gottfred committed:
fixed duplicated code

dnicolalde committed:
Updating labkey image name.

gottfred committed:
Merge pull request #285 from WNPRC-EHR-Services/21.11_fb_purchasing_line_item_numbers

21.11 fb purchasing line item numbers

dnicolalde committed:
Adding websockets to the nginx configuration.

dnicolalde committed:
Adding websocket configuration for second instance of nginx

dnicolalde committed:
Variable for websocket configuration

dnicolalde commented:
Closing this pull request to bring changes to done by WNPRC team
